misc/Makefile.am

```diff
@@ -28,7 +28,7 @@ systemd_DATA  += systemd/cf-postgres.service
 systemd_DATA  += systemd/cf-runalerts.service
 systemd_DATA  += systemd/cf-serverd.service
 
-install-data-local:
+install-data-hook:
```

Comment:
So what's the difference?

Reply:
hook is guaranteed to be after all the other parts of the install-data target. It seems like install-data-local was running before the service files were installed.